### PR TITLE
Adding option to print offset-distance

### PR DESCRIPTION
**Pull request by 88manpreet**
E.g.: 

Cluster name: abc, consumer group: def
Topic Name: xyz
	Partition ID: 0
		High Watermark: 2488
		Low Watermark: 2484
		Current Offset: 125
		Offset Distance: 2363 (Percentage behind: 0.95%)

### CODE REVIEW

File: kafka_utils/kafka_consumer_manager/commands/offset_get.py [hunk 1]
```diff
@@ -148,3 +150,16 @@ def print_output(cls, consumer_offsets_metadata, watermark_filter):
                             current=metadata_tuple.current
                         )
                     )
+                if watermark_filter == "all" or watermark_filter == "distance":
+                    if metadata_tuple.highmark > 0:
+                        per_behind = float(metadata_tuple.highmark - metadata_tuple.current) / \
```
Comment (yzernik) [inline kafka_utils/kafka_consumer_manager/commands/offset_get.py]: Would this be the ratio behind? For the percentage you need to multiply by 100.

Also, do we need the percentage behind?

File: kafka_utils/kafka_consumer_manager/commands/offset_get.py [hunk 2]
```diff
@@ -148,3 +150,16 @@ def print_output(cls, consumer_offsets_metadata, watermark_filter):
...
+                            metadata_tuple.highmark
+                    else:
+                        per_behind = 0.0
+                    print(
+                        "\t\tOffset Distance: {distance} (Percentage behind: "
```
Comment (ecanzonieri) [inline kafka_utils/kafka_consumer_manager/commands/offset_get.py]: How does this look like in json format?

Comment (88manpreet) [inline kafka_utils/kafka_consumer_manager/commands/offset_get.py]: Json example: 

[
  {
    "topic": "scribe.devc.yp_page_views",
    "partition": 0,
    "current": 121,
    "highmark": 124,
    "lowmark": 124,
    "offset_distance": 3,
    "percentage_distance": 2.42
  }
]

New console output: 
Topic Name: scribe.devc.yp_page_views
	Partition ID: 0
		High Watermark: 124
		Low Watermark: 124
		Current Offset: 121
		Offset Distance: 3
		Percentage Distance: 2.42%

File: kafka_utils/kafka_consumer_manager/commands/offset_get.py (hunk 1 above)
Comment (88manpreet) [inline kafka_utils/kafka_consumer_manager/commands/offset_get.py]: Updated.

File: kafka_utils/kafka_consumer_manager/commands/offset_get.py [hunk 3]
```diff
@@ -148,3 +160,26 @@ def print_output(cls, consumer_offsets_metadata, watermark_filter):
                             current=metadata_tuple.current
                         )
                     )
+                if watermark_filter == "all" or watermark_filter == "distance":
+                    per_distance = cls.percentage_distance(metadata_tuple.highmark, metadata_tuple.current)
```
Comment (ecanzonieri) [inline kafka_utils/kafka_consumer_manager/commands/offset_get.py]: Why do we need to cast to int? Can't we cast to int once here when we invoke the function instead than performing the cast multiple times inside percentage_distance()?

Comment (88manpreet) [inline kafka_utils/kafka_consumer_manager/commands/offset_get.py]: updated

**Pull request merged**